app/tests/spec/views/mixins/settings-mixin.js

```diff
@@ -40,9 +40,9 @@ define([
 
     beforeEach(function () {
       relier = new Relier();
-      notifications = new Notifications();
+      notifier = new Notifications();
```

Comment:
Just to reiterate the earlier point, whenever I see `notifier = new Notifications()` now it confuses me.